docs/ref/settings.txt

```diff
@@ -52,9 +52,14 @@ A list of all the people who get code error notifications. When
 is configured in :setting:`LOGGING` (done by default), Django emails these
 people the details of exceptions raised in the request/response cycle.
 
-Each item in the list should be a tuple of (Full name, email address). Example::
+Each item in the list should be an email address string. Example::
 
-    [("John", "john@example.com"), ("Mary", "mary@example.com")]
+    ADMINS = ["john@example.com", '"Ng, Mary" <mary@example.com>']
+
+
+.. versionchanged:: 6.0
+
+    In older versions, required a list of (name, address) tuples.
```

Comment:
I think we should add this note to MANAGERS as well

Reply:
Done